### DISCUSSION TITLE
Annotations of the detections - layout

### DISCUSSION THREAD

**Discussion by Blyuger** [rafaelpadilla/review_object_detection_metrics#19] [Q&A] [answered]
Hi,

I have a question about the given format in review_object_detection_metrics/data/database/dets/abs_xyx2y2.

I am trying to use this format (as i noticed in  review_object_detection_metrics/data/database/dets/abs_xyx2y2):

**class_id, confidence, left, right, top, bottom.**

Here goes the annotation of my detection data:

![layout](https://user-images.githubusercontent.com/13035794/109185701-03936a80-776f-11eb-897f-9c2cf3fe8d62.PNG)

And I selected the option: "(*) <class_id> <confidence> <left> <top> <right> <bottom> (ABSOLUTE)"

Is that right?

Because I formated my detection annotations like this, but when Ì try to run I received the following message in the dialog:

**"No detection of the selected type was found in the folder.
Check if the selected type corresponds to the files in the folder and try again."**

![layout](https://user-images.githubusercontent.com/13035794/109184887-2ec98a00-776e-11eb-9097-cb403dbc6ea3.PNG)

I am using the voc.names (as you do) with just one class_id (mango that is the class 0) because I am predicting mangoes.

Could you help me?

**Comment by rafaelpadilla** [comment, marked as answer]
Hi @Blyuger ,

You annotations are not in ABSOLUTE values. Neither in RELATIVE values, as the coordinates should have values between 0 and 1.

The problem with your annotion is that your bounding box coordinates (17.73, 28.54, 71.39, 96.63) does not match the format (left, top, width, height). Your coordinates are not in absolute values. Note that your bounding box cannot be 17.73 pixels right from the origin, nor 71.39 pixels wide.

You should either change your files with bounding box coordinates to absolute INTEGER values, as in:
`18 29 71 97` or `17 28 71 96`

or fill the decimal places with zeros as the files in the folder database/dets/abs_xyx2y2. So your coordinates should be:
`18.00 29.00 71.00 97.00` or `17.00 28.00 71.00 96.00`

Rafael
